spec/spec_helper.rb

```diff
@@ -17,6 +17,8 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  require 'simplecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.